packages/unchained-client/src/ethereum/parser/constants.ts

```diff
@@ -6,3 +6,4 @@ export const THOR_ROUTER_CONTRACT_ROPSTEN = '0xefA28233838f46a80AaaC8c309077a9ba
 export const FOXY_STAKING_CONTRACT = '0xee77aa3Fd23BbeBaf94386dD44b548e9a785ea4b'
 export const WETH_CONTRACT_MAINNET = '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2'
 export const WETH_CONTRACT_ROPSTEN = '0xc778417E063141139Fce010982780140Aa0cD5Ab'
+export const UNI_V2_STAKING_REWARDS = '0x212ebf9FD3c10F371557b08E993eAaB385c3932b'
```

Comment:
```suggestion
export const UNI_V2_FOX_STAKING_REWARDS = '0x212ebf9FD3c10F371557b08E993eAaB385c3932b'
```

Reply:
Maybe even more specifically `UNI_V2_FOX_STAKING_REWARDS_V3` which I believe is the address being referenced